docs/source/content/supervised/logistic_regression.rst

```diff
@@ -5,52 +5,132 @@ Logistic Regression
 
-=====  ======   ======= 
-  Studying      Success 
+=====  =======  =======
+  Studying      Success
 --------------  -------
-Hours  Focused  Pass? 
-=====  ======   =======
-1      False    True 
-3      False    True 
-0.5    True     False 
-2      False    True 
-=====  ======   =======
+Hours  Focused  Pass?
+=====  =======  =======
+1      False    False
+3      False    True
+0.5    True     False
+2      False    True
+=====  =======  =======
 
-Notice that the student's success is determined by the inputs and the value is binary, so Logistic Regression will work well for this scenario.
+Notice that the student's success is determined by the inputs and the value is
+binary, so logistic regression will work well for this scenario.
 
 
 How does it work?
 -----------------
 
-Logistic regression works using a linear combination of inputs, so multiple information sources can govern the output of the model.  The parameters of the model are the weights of the various features, and represent their relative importance to the result.
+Logistic regression works using a linear combination of inputs, so multiple
+information sources can govern the output of the model.
+The parameters of the model are the weights of the various features, and
+represent their relative importance to the result.
+In the equation that follows, you should recognize the formula used in linear regression.
+Logistic regression is, at its base, a transformation from a linear predictor
+to a probability between 0 and 1.
 
-.. figure:: _img/Logistic_Eq.svg
+.. figure:: _img/WikiLogisticEQ.svg
    :alt: Equation
 Ref: https://en.wikipedia.org/wiki/Logistic_regression
 
-The beta values are the weights and x values are the variable inputs.  This formula gives the probability that the input belongs to Class B, which is the output of the function.
+As in linear regression, the beta values are the weights and x values are the variable inputs.
+This formula gives the probability that the input belongs to Class B, which
+is the goal of the logistic regression model.
+
+
+Multinomial logistic regression
+-----------------
+
+Until now, we've been discussing the situation where there are exactly
+two distinct outputs, for example a pass or a fail.
+But, what if there were more than two possible outputs?
+What about the number classification example, where the output can be any digit from 0 to 9?
+
+Well, there is a way to handle that with logistic regression.
+When using the scikit-learn library, as the example code does, the facility is already there.
+With scikit-learn, you can use the multinomial mode and supply any number of
+classes in the training data.
+You can think of the method as creating multiple models and comparing their
+probabilities, but the exact details_ are beyond the scope of this course.
+
+.. _details: https://en.wikipedia.org/wiki/Multinomial_logistic_regression
+
+Code
+-----------------
+
+Check out the example_ for logistic regression in our repository.
```

Comment:
This link is never defined so it just leads back to this page instead of the code

Reply:
Should be fixed in a7fc1f2